src/server/api/routes/users/v2/follow.js

```diff
@@ -14,7 +14,7 @@ const router = Router();
 
 router.get('/followees', authenticateV2Login, async (req, res, next) => {
   try {
-    const { user } = req.session;
+    const user = req.query?.user || req.session.user;
```

Comment:
Should not replace `req.session.user` in any api that use `authenticateV2Login`, otherwise anyone can access other's information (in this case, followees and pastFollowees) as long as they are logged in.

Should make a new API for suggestion that does not leak followee information of other user